pkg/logging/logger.go

```diff
@@ -16,32 +15,12 @@ const (
 	LogFormatJSON   = "json"
 )
 
-// NewLogger returns a log.Logger that prints in the provided format with a UTC
```

Comment:
nit: We should have the comment for `godoc` since it's an exposed function.

Reply:
My bad, not sure what made me delete this. Added it back and updated it with the current state.